interfaces/forward_auth/v0/README.md

```diff
@@ -0,0 +1,58 @@
+# `forward_auth`
```

Comment:
Should this be called `traefik_forward_auth` -- it seems like this is specific enough that it's unlikely to be applicable to any other setup?

Reply:
It is possible that the interface will be reused for nginx - do you think we should keep them separate?